docs/pages/getting-started/5-irma-contract.rst

```diff
@@ -44,6 +44,8 @@ The domain must be configured on the Nuts node:
 The Nuts APIs used for signing will embed this URL in the QR code shown to the user.
 The javascript in the frontend will also use this URL (exposed via the QR code) to check the status of the signing session.
 Therefore the domain which serves the frontend must be able to do requests to that domain.
+The above URL will be expanded to ``https://example.com/public/auth/irmaclient`` as IRMA path.
+The IRMA ``session`` API will be mounted at that base path.
```

Comment:
It's not clear to me if this API is part of Nuts or IRMA. 
Maybe add link to https://irma.app/docs/api-irma-server/ or point to IRMA in some other way.